content/applications/finance/accounting/bank/loans.rst

```diff
@@ -0,0 +1,72 @@
+:show-content:
+
+================
+Loans management
+================
+
+Odoo's loan management gives a comprehensive list of all loans undertaken by your company in order
+to maintain a holistic and forecasted view of upcoming due dates (e.g., cash forecast). Set up
+amortization schedules—or import them—and let Odoo automatically handle monthly interest and
+principal adjustments so that your financial reports are always accurate with minimal effort.
+
+Create a new loan
+-----------------
+
+Create a new loan by going to :menuselection:`Accounting --> Accounting --> Loans`. When creating a
+new loan, there are three options for how to create amortization schedules:
+
+- importing it from a supported file
+- calculating it from multiple input values (e.g. the :guilabel:`Amount Borrowed`, the
+  :guilabel:`Duration`, etc.) using the :guilabel:`Compute` button.
+- manually filling in the lines of the schedule
```

Comment:
```suggestion
- importing it from a supported file;
- calculating it from multiple input values (e.g., the :guilabel:`Amount Borrowed`, the
  :guilabel:`Duration`, etc.) using the :guilabel:`Compute` button;
- manually filling in the lines of the schedule.
```

Reply:
This one was my mistake. I suggested removing the semi-colons but looking at it again, it's better with. Sorry about that one @RicGR98 !